oauthproxy.go

```diff
@@ -845,23 +852,23 @@ func (p *OAuthProxy) Authenticate(rw http.ResponseWriter, req *http.Request) int
 	}
 
 	if session != nil && session.IsExpired() {
-		log.Printf("%s removing session. token expired %s", remoteAddr, session)
+		logger.Printf("Removing session; token expired %s", session)
 		session = nil
 		saveSession = false
 		clearSession = true
 	}
 
 	if saveSession && !revalidated && session != nil && session.AccessToken != "" {
 		if !p.provider.ValidateSessionState(session) {
-			log.Printf("%s removing session. error validating %s", remoteAddr, session)
+			logger.Printf("Removing session; error validating %s", session)
```

Comment:
Same as above? Colon?
```suggestion
			logger.Printf("Removing session: error validating %s", session)
```